src/Testing/src/xunit/SkipOnHelixAttribute.cs

```diff
@@ -52,7 +52,7 @@ private bool ShouldSkip()
 
             var targetQueue = GetTargetHelixQueue().ToLowerInvariant();
 
-            if (Queues.Contains("All.OSX") && targetQueue.StartsWith("osx"))
+            if (Queues.Contains("All.OSX") && targetQueue.StartsWith("osx", StringComparison.OrdinalIgnoreCase))
```

Comment:
```suggestion
            if (Queues.Contains("All.OSX") && targetQueue.StartsWith("osx", StringComparison.Ordinal))
```

That would be the equivalent to the previous check

Reply:
I'll let this be. This is test-only code so it doesn't make a difference.

Reply:
@HaoK actually asked to change this to OrdinalIgnoreCase as part of this review.